WooCommerce/Classes/Tools/Zendesk/ZendeskManager.swift

```diff
@@ -810,6 +858,12 @@ private extension ZendeskManager {
         return logText
     }
 
+    /// Get the System Status Report as a formatted String that we can use in the support request
+    ///
+    func getSystemStatusReport() -> String {
```

Comment:
We don't need `self` here right? And we perhaps don't even need this function at all? what if use directly the `systemStatusReport` property wherever it is called?

Reply:
You're right, we can use the property directly. Changed on 1ae3db12553bb6f1f9fd949bc0538fd8d612e923